keps/sig-node/1972-kubelet-exec-probe-timeouts/kep.yaml

```diff
@@ -15,16 +15,17 @@ approvers:
   - "@derekwaynecarr"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: stable
+stage: implemented
 
```

Comment:
line 8
> status: implemented
status is already implemented here

Reply:
Thanks! I have modified the file again.